### DISCUSSION TITLE
Disable moving backwards

### DISCUSSION THREAD

**Discussion by 3goats** [simeydotme/svelte-range-slider-pips#100] [Formatting / Styling]
Hi - Thanks for a great utility.

I'm going to re-phrase this question. 

Is it possible to disable clicks on previous steps within the range slider. For example if you increment the slider by clicking the steps from left to right, disable the previous step. 

I want to use the slider to control steps within an animated SVG and don't want handler having to reverse the timeline if the user clicks on a previous step. 

I probably also need to disable the next step until each animation completes. 

So the question is really - is there a way to programatically enable disable the clicking for each of the seps in the slider ?

**Comment by simeydotme** [comment]
not sure I understand...

but the short answer is no.

the long answer is if you're using Svelte/Vue/React you can treat the slider as a proxy value, and create another variable to hold the real state, and then using the "onchange" event; every time the proxy value is changed you can check that it passes your conditions, and if it does, update your real state, otherwise reset the proxy value to it's previous state.

hope that's helpful :)
